module/VuFind/tests/integration-tests/src/VuFindTest/Mink/CartTest.php

```diff
@@ -670,17 +655,18 @@ public function testCartExport()
      */
     public function testCartExportToThirdParty()
     {
+        $exportUrl = $this->getVuFindUrl('/Content/export_test');
```

Comment:
Does it matter that this is going to return a 404 error? Or am I missing something?

Reply:
It doesn't matter since we just check that a window with the expected url was opened. I added a comment to clarify it.